lib/read.js

```diff
@@ -824,7 +824,8 @@ Reader.prototype.fromXML = function(xml, options, done) {
 
     return uriMap;
   }, {});
-
+  // add default xml ns
+  uriMap['http://www.w3.org/XML/1998/namespace'] = 'xml';
```

Comment:
Can we initialize the XML mapping on line 826, i.e. feed it as the default mapping into reduce:

```javascript
var uriMap = model.getPackages().reduce(function(uriMap, p) {
    uriMap[p.uri] = p.prefix;

    return uriMap;
  }, {
    'http://www.w3.org/XML/1998/namespace': 'xml'
  });
```